clients/ui/frontend/src/app/pages/modelCatalog/components/ValidatedModelCard.tsx

```diff
@@ -0,0 +1,247 @@
+            </FlexItem>
+          </Flex>
+          <div style={{ marginTop: '8px' }}>
+            <Flex alignItems={{ default: 'alignItemsCenter' }} gap={{ default: 'gapSm' }}>
+              <div
+                data-testid="validated-model-status-indicator"
+                className={text.textColorStatusSuccess}
+                style={{
+                  width: '8px',
+                  height: '8px',
+                  backgroundColor: 'var(--pf-t--global--palette--green-400)',
+                  borderRadius: '50%',
+                }}
+              />
+              <Content component={ContentVariants.small} className={text.textColorStatusSuccess}>
+                Success
+              </Content>
+            </Flex>
+            <Link to={catalogModelDetailsFromModel(model.name, source?.id)}>
+              <Button
+                data-testid="model-catalog-detail-link"
+                variant="link"
+                tabIndex={-1}
+                isInline
+                style={{ fontSize: '1.25rem', fontWeight: 'bold' }}
+              >
+                {truncate ? (
+                  <span>{getModelName(model.name)}</span>
+                ) : (
+                  <span>{getModelName(model.name)}</span>
+                )}
+              </Button>
+            </Link>
+          </div>
+        </CardTitle>
+      </CardHeader>
+      <CardBody>
+        <div>
+          <div style={{ paddingLeft: '8px', marginBottom: '16px' }}>
+            <Flex
+              alignItems={{ default: 'alignItemsCenter' }}
+              justifyContent={{ default: 'justifyContentSpaceBetween' }}
+            >
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                gap={{ default: 'gapSm' }}
+                wrap="nowrap"
+              >
+                <ChartBarIcon />
+                <span
+                  data-testid="validated-model-accuracy"
+                  style={{ fontSize: '1.25rem', fontWeight: 'bold' }}
+                >
+                  {metrics.accuracy}%
+                </span>
+              </Flex>
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                gap={{ default: 'gapSm' }}
+                wrap="nowrap"
+              >
+                <span className="pf-v5-c-content pf-m-small">Average accuracy</span>
+                <Tooltip
+                  content="Average accuracy
+
+The weighted average of normalized scores from all benchmarks. Each benchmark is normalized to a 0-100 scale. All normalized benchmarks are then averaged together."
+                  position="top"
+                  className="pf-v6-c-tooltip"
+                >
+                  <HelpIcon style={{ cursor: 'help' }} />
+                </Tooltip>
+              </Flex>
+            </Flex>
+          </div>
+
+          {/* Hardware Configuration and Latency */}
+          <div style={{ paddingLeft: '8px', marginBottom: '16px' }}>
```

Comment:
As above, see what happens if you just don't use custom styles here and use Stack/StackItem to separate these vertical areas.